packages/framework/client-logger/app-insights-logger/package.json

```diff
@@ -136,7 +137,8 @@
 			"build:docs": {
 				"dependsOn": [
 					"...",
-					"api-extractor:commonjs"
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
```

Comment:
Does it depend on `"esnext`? Is there an exclusive access issue?

Reply:
Yes, this task needs to depend on both. This will get cleaned up once all packages are using this new task definition and we can update the root fluidBuild.config file.